### ISSUE TITLE
Central Management - On the Computer details page, allow editing groups computer is a member of

### ISSUE THREAD

**Issue by ottopodo** [chocolatey/chocolatey-licensed-issues#341]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

Adding a user to a single group or to multiple groups in CCM is a complex operation. 

### Describe The Solution. Why is it needed?

Add a button on Computer detail page to add a computer to an existing group.

### Related Issues

* [GitLab](https://gitlab.com/chocolatey/central-management/choco-licensed-management-ui/-/issues/1089)

**Comment by pauby** [comment]
@ottopodo The Solution you mentioned, and the issue title don't line up. Is this for adding users or computers?

Can you go back and add more information in to the original issue comment above so that we are very clear on what it is you are asking? Screenshots would be a great addition.

**Comment by ottopodo** [comment]
Modal
![immagine](https://user-images.githubusercontent.com/8350180/235132653-b13d226b-6fe9-4eb5-b6ed-cdf02d08eaf0.png)

**Comment by ottopodo** [comment]
![screenshot_modal](https://user-images.githubusercontent.com/8350180/235133236-7371cf74-a6d4-4e7f-b7df-7a6edcb1df11.png)

**Comment by ottopodo** [comment]
![Screenshot_main](https://user-images.githubusercontent.com/8350180/235137880-dae61e40-e0be-4859-a912-bb26c22cecdf.png)

**Comment by gep13** [comment]
@ottopodo thank you for providing those mock up screenshots, I think they help to illustrate what is being requested here, but just to make sure that we are all on the same page...

Currently, on the Groups page, it is possible to Add/Edit an existing Group, by moving either Computers or other Groups, from the left hand list box to the right hand list box:

<img width="1845" alt="image" src="https://user-images.githubusercontent.com/1271146/235163235-3681be04-9b3a-4cf8-bc74-1045a1b64cce.png">

<img width="1835" alt="image" src="https://user-images.githubusercontent.com/1271146/235163322-5dc6cc61-dc52-4dc8-992a-24ec8c38fc4b.png">

Currently, in the Computer Details page, we have the following:

<img width="1843" alt="image" src="https://user-images.githubusercontent.com/1271146/235163514-94a648f9-c090-46da-9c09-2db62e34f99e.png">

You are suggesting that we add a new `Edit Groups` button to the Computer Details page, which is what you are showing here https://github.com/chocolatey/chocolatey-licensed-issues/issues/341#issuecomment-1527434007 and then, when the user clicks the `Edit Groups` button, a new modal (which is what you are showing here https://github.com/chocolatey/chocolatey-licensed-issues/issues/341#issuecomment-1527405450), to allow moving different Groups between the left and right listboxes, to change which Groups a specific computer is a member of.

Have I got that right?

The only thing I would suggest is that in this Modal https://github.com/chocolatey/chocolatey-licensed-issues/issues/341#issuecomment-1527405450 the Title would be `Edit Computer` rather than `Edit Group`.

Want to make sure that we are on the same page, so that we can figure out how feasible this is, and what work would be required to make it work.

**Comment by ottopodo** [comment]
Yes correct, I create screenshot from groups management and I didn't change title.
